exoctk/exoctk_app/app_exoctk.py

```diff
@@ -460,7 +460,7 @@ def contam_visibility():
     if form.mode_submit.data:
 
         # Update the button
-        if form.inst.data != 'NIRISS':
+        if (form.inst.data == 'MIRI') or (form.inst.data == 'NIRSpec'):
```

Comment:
Why did we decided to disable the form for NIRSpec too?

Reply:
we don't have plans to use the contamination calculator for NIRSpec currently @mfixstsci